### ISSUE TITLE
Pixelated taskbar icon

### ISSUE THREAD

**Issue by Lilkedus** [getinsomnia/insomnia#3099]
The icon's quality, in the taskbar, is very low.

Here is a screenshot:
![image](https://user-images.githubusercontent.com/71512866/108596772-3a950500-7398-11eb-8ea1-4be815e95d07.png)

**Comment by tbolt** [comment]
This also stuck out to me. The desktop icon is sharp but the taskbar really sticks out as blurry
![icon](https://user-images.githubusercontent.com/2279260/113423907-4dd6bf80-939d-11eb-9136-d0c5073183bc.png)

**Comment by tbolt** [comment]
👀 
![Screenshot 2021-04-02 105124](https://user-images.githubusercontent.com/2279260/113426375-5fba6180-93a1-11eb-9d10-6e776a69bb60.png)

**Comment by dimitropoulos** [comment]
Thanks for the report.  I noticed this too (on linux).

Does anyone happen to know, off hand, how we might address this?  I don't know off the top of my head, but of course we could investigate.

**Comment by wdawson** [comment]
I suspect our `.ico` and `.icns` files (I think they're [here](https://github.com/Kong/insomnia/tree/develop/packages/insomnia-app/app/icons)) don't have high enough resolutions included, though it seems fine on macOS. 

Seems like this might help: https://www.electron.build/icons

**Comment by filfreire** [comment]
Hi @Lilkedus are you still able to reproduce this issue in our [latest Insomnia stable version](https://github.com/Kong/insomnia/releases/tag/core%402022.4.2)?

**Comment by Lilkedus** [comment]
> Hi @Lilkedus are you still able to reproduce this issue in our [latest Insomnia stable version](https://github.com/Kong/insomnia/releases/tag/core%402022.4.2)?

No, I am not. I don't use windows anymore. Insomnia on my mac is no longer blurry. Sorry

**Comment by skrachkovsky** [comment]
The issue still reproduces on Windows 11. 4k 3840x2160 display 250% scale (15.6-inch laptop). The bug shows up when the application is running (pinned icons are OK).

**Comment by hadyawayda** [comment]
Same issue as well until you pin the application I guess it becomes a normal icon!
